index.html

```diff
@@ -0,0 +1,322 @@
+        <ul class="sid-cons">
+          <li class="p">Portoflio</li>
+          <li class="a">About</li>
+          <li class="c">Contact</li>
+          <li class="material-symbols-outlined">menu</li>
+        </ul>
+      </nav>
+    <div class="headline-container">
+      <div class="items1">
+        <h1 class="items item1">I'm Gasangwa Glad to see you !</h1>
+        <p class="items item3">
+          I am a software developer. I can help you build a product, feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need to be coded, don't hesitate to
+          contact me.
+        </p>
+        <div>
+          <h4 class="contact">LET'S CONNECT</h4>
+          <ul class="icons">
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/twitter.png"
+                  alt="twitter-icon"
+                  class="twitter"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/linkedin.png"
+                  alt="linkedin-icon"
+                  class="linkedin"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/medium.png"
+                  alt="medium-icon"
+                  class="medium"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/github.png"
+                  alt="github-icon"
+                  class="github"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/angellist.png"
+                  alt="angellist-icon"
+                  class="angellist"
+              /></a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    </div>
+    <!--html of the middle section-->
+    <section class="middle-section">
+      <div class="card1">
+        <div class="snapshoot">
+          <img src="./images/screenshot1.png" alt="screenshot" class="responsive"/>
+        </div>
+        <div class="word-title">
+          <div class="w-title">
+            <h2 class="name">Thomas</h2>
+            <ul class="canop">
+              <li>CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div class="p-text">
+            <p class="word">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div>
+            <ul class="languages">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <button type="submit" class="tap">See Project</button>
```

Comment:
- [ ] When hovering over the buttons the text on the buttons is not visible, please make sure to apply the correct text color while hovering over the buttons. (Please fix this issue for all the buttons).

### Your design

![image](https://user-images.githubusercontent.com/83803414/166200699-b38c7ed9-6760-4d64-9e89-3fe3ff7ae04d.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/166200732-45c9b387-14d9-4bb6-bfc8-d6263f867d34.png)